.github/workflows/os_comp.yml

```
@@ -130,6 +133,15 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Get zig
+        if: ${{ contains(matrix.cfg.CC, 'zig') }}
+        run: |
+          ZIGVER=${{ matrix.cfg.VERSION }}
+          wget https://ziglang.org/download/$ZIGVER/zig-$ZIGVER.tar.xz
+          tar xf "zig-$ZIGVER.tar.xz"
+          cd "zig-$ZIGVER"
+          echo "PATH=$PWD:$PATH" >> $GITHUB_ENV
```

This should use `GITHUB_PATH`:
```suggestion
          echo "${PWD}" >> $GITHUB_PATH
```

---

Thanks @QuLogic . I think now after the changes I just made, zig will probably be dumped into a pre-existing tools directory that's added to the path a little later.